bin/mkreleaselog

```diff
@@ -6,8 +6,30 @@ export GOPATH="$(go env GOPATH)"
 
```

Comment:
Not everybody has go-ipfs pulled down as an actual dep

```suggestion
! [[ -d "$GOPATH/src/github.com/ipfs/go-ipfs" ]] \
&& echo 'You must leave the current repository dir and `go get -v github.com/ipfs/go-ipfs` before using this script' \
&& exit 1
```

Reply:
Well, this script assumes you're using a GOPATH and won't work without it, unfortunately.